tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -1648,5 +1648,75 @@ describe("Service Form Modal", function() {
         });
       });
     });
+
+    context("Multi-container (pod)", function() {
+      beforeEach(function() {
+        cy
+          .get(".menu-tabbed-item-label")
+          .eq(0)
+          .click()
+          .get(".menu-tabbed-view h2")
+          .contains("Service");
+      });
+
+      it("Should add new container", function() {
+        cy.get(".menu-tabbed-view .button.button-primary-link").eq(3).click();
+      });
+
+      it("Should contain two containers navigation under Services tab", function() {
+        cy.get(".menu-tabbed-view .button.button-primary-link").eq(3).click();
+
+        cy
+          .get(".menu-tabbed-item-label")
+          .eq(0)
+          .siblings()
+          .should("to.have.length", 2);
+      });
+
+      it("Should be right aligned of the parent", function() {
+        cy.get(".menu-tabbed-view .button.button-primary-link").eq(3).click();
+
+        cy
+          .get(".menu-tabbed-item-label")
+          .eq(0)
+          .siblings()
+          .each(function($element) {
+            const $parentLeftPosition = $element
+              .parent()[0]
+              .getBoundingClientRect().left;
+            const $elementLeftPosition = $element[0].getBoundingClientRect()
+              .left;
+            const threshold = 10;
+            const difference = $elementLeftPosition - $parentLeftPosition;
+
+            expect(difference >= threshold).to.equal(true);
+          });
+      });
+    });
+
+    context("Multi-container (pod) Review & Run", function() {
+      beforeEach(function() {
+        // Fill in SERVICE ID
+        cy.get('.form-control[name="id"]').clear().type("/test-review-and-run");
+
+        // Add a second container
+        cy.get(".menu-tabbed-view .button.button-primary-link").eq(3).click();
+
+        // Click review and run
+        cy
+          .get(".modal-full-screen-actions")
+          .contains("button", "Review & Run")
+          .click();
+      });
+
+      it("Should add new container", function() {
```

Comment:
Test description doesn't match the content. Easiest would be to change the description to something like "Should contain two containers" since that it exactly what this specific test is verifying.